### DISCUSSION TITLE
useBottomTabBarHeight incorrect height?

### DISCUSSION THREAD

**Discussion by tamis-laan** [react-navigation/react-navigation#9723] [Q&A] [answered]
I'm wondering If I'm doing something wrong or if this is a bug. I have a FlatList inside a bottomTabBar navigation. I want each item in the FlatList to fill up the screen. So I calculate the width and height of a FlatList item as followed:

``` javascript
  import { Dimensions } from 'react-native'
  import { useBottomTabBarHeight } from '@react-navigation/bottom-tabs';

  const width  = Dimensions.get('window').width
  const height = Dimensions.get('window').height-useBottomTabBarHeight()
```
The function `useBottomTabBarHeight()` returns a value of `49.0909` however this does not produce the wanted result. By manually fudging the values I get value of `73` for the height of the bottom tab bar. 

Is there something I'm missing here?

EDIT:
It looks as if the android global navigation buttons are not included inside `Dimentions.get(screen).height` which they should right !?

**Comment by tamis-laan** [comment, marked as answer]
Ah for anyone dealing with the same problem it turns out it was the size of the status bar in android that was the culprit:

``` javascript
  import { Dimensions, StatusBar } from 'react-native'
  import { useBottomTabBarHeight } from '@react-navigation/bottom-tabs';

  const width  = Dimensions.get('window').width
  const height = Dimensions.get('window').height-useBottomTabBarHeight()-StatusBar.currentHeight
```

**Reply by gauravstanza** [reply]
can we use some kind of conditional base here as for most of the devices useBottomTabBarHeight() works just fine but in few of them adding StatusBar.currentHeight to it works.
